src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -0,0 +1,187 @@
+                    Debug.Fail("Failed to resolve conflict for " + aggregate.RuleSetFilePath, ex.ToString());
+                }
+            }
+
+            return conflicts;
+        }
+
+        private RuleSetAggregate[] GetAggregatedSolutionRuleSets()
+        {
+            var solutionBinding = this.serviceProvider.GetService<ISolutionBinding>();
+            solutionBinding.AssertLocalServiceIsNotNull();
+
+            BoundSonarQubeProject bindingInfo = solutionBinding.ReadSolutionBinding();
+            if (bindingInfo != null)
+            {
+                var projectSystem = this.serviceProvider.GetService<IProjectSystemHelper>();
+                projectSystem.AssertLocalServiceIsNotNull();
+
+                var ruleSetInfoProvider = this.serviceProvider.GetService<ISolutionRuleSetsInformationProvider>();
+                ruleSetInfoProvider.AssertLocalServiceIsNotNull();
+
+                var fileSystem = this.serviceProvider.GetService<IFileSystem>();
+                fileSystem.AssertLocalServiceIsNotNull();
+
+                var projectRuleSetAggregation = new Dictionary<string, RuleSetAggregate>(StringComparer.OrdinalIgnoreCase);
+
+                // TODO: we need to filter the same as in binding workflow:
+                // it requires various changes and will be done separately.
+
+                foreach (Project project in projectSystem.GetSolutionProjects())
+                {
+                    string suffix = SolutionBindingOperation.GetProjectRuleSetSuffix(ProjectBindingOperation.GetProjectGroup(project));
+                    string baselineRuleSet = ruleSetInfoProvider.CalculateSolutionSonarQubeRuleSetFilePath(bindingInfo.ProjectKey, suffix);
+
+                    if (!fileSystem.FileExist(baselineRuleSet))
+                    {
+                        this.WriteWarning(Strings.ExpectedRuleSetNotFound, baselineRuleSet, project.FullName);
+                        continue;
+                    }
+
+                    foreach (RuleSetDeclaration declaration in ruleSetInfoProvider.GetProjectRuleSetsDeclarations(project))
+                    {
+                        string projectRuleSet = CalculateProjectRuleSetFullPath(fileSystem, project, declaration);
+
+                        if (string.IsNullOrWhiteSpace(projectRuleSet))
+                        {
+                            // Use the original property value to attempt to load the rule set with the directories information, during FindConflicts
+                            projectRuleSet = declaration.RuleSetPath;
+
+                            // We do want to continue and add this rule set rather than skip over it since it maybe the case that
+                            // the user moved it to some other location which is relative to the rule set directories property
+                            // and we will be able to find it during rule set conflicts analysis.
```

Comment:
There's no point in doing this if the original value is absolute.
I'd be tempted to push the "if-null" logic into _CalculateProjectRuleSetFullPath_ i.e. put all of the relevant logic there, along with appropriate comments.